addons/fleet/models/fleet_vehicle_model.py

```diff
@@ -36,13 +37,35 @@ class FleetVehicleModelBrand(models.Model):
     image_128 = fields.Image("Logo", max_width=128, max_height=128)
     model_count = fields.Integer(compute="_compute_model_count", string="", store=True)
     model_ids = fields.One2many('fleet.vehicle.model', 'brand_id')
+    car_count = fields.Integer(compute="_compute_vehicle_count")
+    bike_count = fields.Integer(compute="_compute_vehicle_count")
 
     @api.depends('model_ids')
     def _compute_model_count(self):
         Model = self.env['fleet.vehicle.model']
         for record in self:
             record.model_count = Model.search_count([('brand_id', '=', record.id)])
 
+    @api.depends('model_ids')
+    def _compute_vehicle_count(self):
+        car_dict = defaultdict(int)
+        bike_dict = defaultdict(int)
+
+        datas = self.env['fleet.vehicle.model'].read_group(
```

Comment:
Could you rename `datas` into something else? It's not a proper English word :sweat_smile: